pages/donate.md

```diff
@@ -7,44 +7,26 @@ teaser: ""
 permalink: "/donate/"
 ---
 
-All of the work done by Lucy Parsons Labs is volunteer funded and directed. Below are a few ways you can help fund transparency in Chicago through our work with either a monthly subscription, a one-time lump sum donation or if you'd like to donate funds to us pseudonymously with Bitcoin. We also have a YouCaring support page where you can donate to us. 
+
+----------------------------
+
+All of the work done by Lucy Parsons Labs is volunteer funded and directed. Below are a few ways you can help fund transparency in Chicago through our work with a one-time donation or if you'd like to donate funds to us pseudonymously with Bitcoin. We also have a YouCaring support page where you can donate to us. 
```

Comment:
How about something simpler like: "All of the work done by Lucy Parsons Labs is volunteer funded and directed.. Below you can fund Lucy Parsons Labs through PayPal, YouCaring, or Bitcoin"?